### DISCUSSION TITLE
github linguist get language statistics for mutiple repos in one folder

### DISCUSSION THREAD

**Discussion by X011ABC** [github-linguist/linguist#5642] [Q&A] [answered]
Hello I am new to github lingust and ruby. i have installed them borht on windows 10 and it is working but does anyone know how I can scan multiple repositories that i put in one folder at once. I tried running github linguist command on the folder where each repo is but it did not not work i have to write the path to each individual repo in the folder which will take some time. 
Any one know how to speedup up that process?

**Comment by lildude** [comment, marked as answer]
Linguist can only analyse one repo at a time so you'll need to iterate over all the repos individually and get the stats for each individually. Assuming you're using WSL, something like this should do the tric: `for d in $(ls -1); do github-linguist $d; done`.
